.travis.yml

```diff
@@ -18,6 +18,9 @@ dist: trusty
 sudo: false
 language: c
 
+git:
+  depth: false
```

Comment:
What's the point of this? From the travis docs `depth: false` removes the `--depth` argument. Why do a full clone when we're not using it?

Reply:
We have to use it, otherwise the VERSION macro wouldn't work, as `git --describe` would fail otherwise.